aws-elasticbeanstalk-terraform/README.md

```diff
@@ -0,0 +1,29 @@
+# SCIM bridge with AWS elastic beanstalk
+
+This is only an example of how you could deploy the 1Password SCIM Bridge to your existing AWS infrastructure. As an advanced form of deployment, this AWS/Terraform example can be integrated into your existing infrastructure. Feel free to modify the deployment to suit your specific needs.
+
+# What are we using
+
+We highly recommend using terraform to create this AWS infrastructure, this help reduce the errors while deploying and maintining AWS infrastructure.
+
+# How to deploy the bridge
+
+1. Have the prerequisite "scimsession" file ready, and have read the [preparation document](../Preparation.md)
```

Comment:
This link seems to be broken, at least in the GitHub UI (is it case sensitive?)

Reply:
Also, does it mention anywhere that I have to put the scimsession file somewhere in particular?

Reply:
That's exactly why I wanted some help.
I'll fix according to your comments.

- Fix link
- add info on where to put the scimsession